src/Scripts/Game/Components/Vehicle/EL_LicensePlateManagerComponent.c

```diff
@@ -0,0 +1,122 @@
+[EntityEditorProps(category: "EL/LicnesePlateManagerComponent", description: "This renders license plates on your vehicle.", color: "0 0 255 255")]
+class EL_LicensePlateManagerComponentClass: ScriptComponentClass
+{
+};
+
+[BaseContainerProps()]
+class EL_LicensePlateComponent
+{
+	[Attribute(uiwidget: UIWidgets.Auto)]
+	ref EL_PointInfo m_Point;
+	
+	EL_LicensePlateEntity m_Object;
+	
+	void ~EL_LicensePlateComponent()
+	{
+		if (m_Object)
```

Comment:
Single line if

Reply:
Those should be fine. Or is there an official guideline against it?

Reply:
No guideline but should be consistent right?